packages/chaire-lib-common/src/services/routing/RoutingResult.ts

```diff
@@ -6,36 +6,63 @@
  */
 import _get from 'lodash/get';
 
-import Preferences from 'chaire-lib-common/lib/config/Preferences';
-import { Route } from 'chaire-lib-common/lib/services/routing/RoutingService';
-import { RoutingOrTransitMode, RoutingMode } from 'chaire-lib-common/lib/config/routingModes';
-import TrError, { ErrorMessage } from 'chaire-lib-common/lib/utils/TrError';
-import { TrRoutingRoute } from 'chaire-lib-common/lib/services/trRouting/TrRoutingService';
+import Preferences from '../../config/Preferences';
+import { Route } from './RoutingService';
+import { RoutingOrTransitMode, RoutingMode } from '../../config/routingModes';
+import TrError, { ErrorMessage } from '../../utils/TrError';
+import { TrRoutingRoute } from '../trRouting/TrRoutingService';
+
+export const pathIsRoute = (path: Route | TrRoutingRoute | undefined): path is Route => {
+    return typeof (path as any).distance === 'number';
+};
 
 // TODO Add a common type to getPath(index)
-export interface RouteCalculatorResult<InputParams> {
+// TODO Have a common type for all results, not requiring the TResultData generic type
```

Comment:
This TODO will be done in the next PR. The `getParams()` is only used in the backend to convert back from the object to the data, but the backend calculation will return the data directly and we won't need to pass by the object.